en/docs/chapter_tree/binary_search_tree.md

```diff
@@ -88,25 +88,25 @@ Assuming we choose the smallest node of the right subtree (the next node in in-o
 === "<4>"
     ![bst_remove_case3_step4](binary_search_tree.assets/bst_remove_case3_step4.png)
 
-The operation of removing a node also uses $O(\log n)$ time, where finding the node to be removed requires $O(\log n)$ time, and obtaining the in-order traversal successor node requires $O(\log n)$ time. Example code is as follows:
+Removing a node also uses $O(\log n)$ time, where finding the node to be removed takes $O(\log n)$ time, and obtaining the in-order traversal successor node takes $O(\log n)$ time. Example code is as follows:
```

Comment:
i think the original is better:
"The operation of removing a node also uses $O(\log n)$ time, where finding the node to be removed requires $O(\log n)$ time, and obtaining the in-order traversal successor node requires $O(\log n)$ time. Example code is as follows:"

yes its wordy, however I think it will be a bit better for newcomers learning DSA.